skbio/io/format/tests/data/fastq_single_seq_illumina1.8

```diff
@@ -0,0 +1,4 @@
+@EAS139:136:FC706VJ:2:2104:15343:197393 1:Y:18:ATCACG
```

Comment:
Same here, if you need help cleaning up your `git` history, come see me or @jairideout later this afternoon.